src/timer/timer_game_calendar.h

```diff
@@ -76,10 +77,12 @@ class TimerGameCalendar {
 	struct TStorage {
 	};
 
-	using Date = int32_t; ///< The type to store our dates in
+	/** The type to store our dates in. */
+	using Date = StrongType::Typedef<int32_t, struct DateTag, StrongType::Explicit, StrongType::Compare, StrongType::Integer>;
+
 	using DateFract = uint16_t; ///< The fraction of a date we're in, i.e. the number of ticks since the last date changeover
 
-	using Year = int32_t; ///< Type for the year, note: 0 based, i.e. starts at the year 0.
+	using Year = StrongType::Typedef<int32_t, struct YearTag, StrongType::Explicit, StrongType::Compare, StrongType::Integer>; ///< Type for the year, note: 0 based, i.e. starts at the year 0.
```

Comment:
```suggestion
	/** Type for the year, note: 0 based, i.e. starts at the year 0. */
	using Year = StrongType::Typedef<int32_t, struct YearTag, StrongType::Explicit, StrongType::Compare, StrongType::Integer>;
```
If a block comment is good enough for Date, it's good enough for this. 😉 

...is your OCD contagious? 😛